src/pages/resource/ResourceMetadata.tsx

```diff
@@ -1,14 +1,18 @@
 import React, { FC } from 'react';
 import { Resource } from '../../types/resource.types';
 import { StyledContentWrapperMedium, StyledSchemaPartColored } from '../../components/styled/Wrappers';
-import { Colors } from '../../themes/mainTheme';
+import { Colors, StyleWidths } from '../../themes/mainTheme';
 import { Chip, Grid, Typography } from '@material-ui/core';
 import { format } from 'date-fns';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
 import { SearchParameters } from '../../types/search.types';
 
 const StyledChip: any = styled(Chip)`
+  max-width: 13rem;
```

Comment:
Vi må kanskje se mer på maks-lengde. Vi kan jo fremdeles registrere lange emneord manuelt. Enten må vi kanskje vise dem fullstendig. eller ha en mouse-over for å vise hele innholdet ? men det jo ikke uu-vennlig
![image](https://user-images.githubusercontent.com/46489468/119647723-e7ac6d00-be20-11eb-8f8e-3bf53ab0ef13.png)

Reply:
Multiline chips